tests/pl/cover_HassOpenCover.yaml

```diff
@@ -27,6 +27,7 @@ tests:
       - "Otwórz rolety w kuchni"
       - "Otwórz wszystkie rolety w kuchni"
       - "Rolety w kuchni otwórz"
+      - "Otwórz całe rolety w kuchni"
```

Comment:
For me correct the correct sentence is "Otwórz rolety w kuchni" or "Otwórz wszystkie rolety w kuchni". I never heard "całe rolety" 😀